src/CoreBundle/private/modules/CMSTreeNodeSelectWYSIWYG/CMSTreeNodeSelectWYSIWYG.class.php

```diff
@@ -13,6 +13,7 @@
 
 /**
  * {@inheritdoc}
+ * @deprecated since 6.3.7 - use NavigationTreeSingleSelectWysiwyg instead
```

Comment:
6.3.8

Reply:
I searched for all "deprecated since 6.3.7" and changed it to "deprecated since 6.3.8" (if they have anything to do with this issue of course)